l10n_it_fatturapa_out/models/account.py

```diff
@@ -49,6 +49,13 @@ def _compute_fatturapa_state(self):
 
     def preventive_checks(self):
         for invoice in self:
+            if not invoice.is_sale_document():
+                raise UserError(
+                    _(
+                        "Impossible to generate XML: not a customer invoice: %s"
+                        % invoice.name
+                    )
```

Comment:
```suggestion
                    _("Impossible to generate XML: not a customer invoice: %s")
                    % invoice.name
```
La sostituzione andrebbe fatta dopo la traduzione e non viceversa, altrimenti Odoo cercherà la traduzione della stringa con già il numero di fattura inserito tipo
```
Impossible to generate XML: not a customer invoice: ACQ/2021/09/0001
```
senza trovarla e l'errore presenterà la stringa non tradotta. Puoi correggere?